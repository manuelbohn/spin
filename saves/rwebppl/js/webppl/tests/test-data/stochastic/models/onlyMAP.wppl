var model = function() {
  return gaussian(1, 1);
};
