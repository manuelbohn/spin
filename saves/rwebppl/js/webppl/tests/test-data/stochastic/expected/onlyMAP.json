{
  "mean": 1,
  "std": 0
}
